packages/cardano-services/test/Asset/CardanoTokenRegistry.test.ts

```diff
@@ -0,0 +1,40 @@
+import { Asset } from '@cardano-sdk/core';
+import { CardanoTokenRegistry } from '../../src/Asset';
+import { dummyLogger } from 'ts-log';
+
+describe('CardanoTokenRegistry', () => {
+  const tokenRegistry = new CardanoTokenRegistry({ logger: dummyLogger });
+  const validSubject = 'f43a62fdc3965df486de8a0d32fe800963589c41b38946602a0dc53541474958';
+
+  it('returns null for unexisting subject', async () => {
```

Comment:
This will return an empty array when using the batch request

Reply:
Rather than an empty array I'm returning an array of `null`s to be able to distinguish the assets with metadata between the ones without in case of requests with mixed good and bad asset ids.
I need confirmation this is a good option.
Thank you

Reply:
Yes, that's a suitable return value.